README.md

```diff
@@ -1,8 +1,58 @@
-# Bifrost
+# DPM
 
-Bifrost is a monorepo starter for fullstack application using Nextjs and Nestjs.
+The World Food Program Cambodia supports the National Committee for Disaster Management (NCDM) and their provincial counterparts (PCDM) on the use of mobile data collection tools to capture information on disasters.
 
-## Getting started
+This DPM project is a form management tool. It provides the ability to manage users and campaigns as it relates to the disaster relief within Cambodia's provinces.
 
-- Follow the [installation](./docs/installation.md) docs
+### Getting Started
 
+## Installation
+
+- [Install fnm](https://github.com/Schniz/fnm#installation) or another node version manager (I _highly_ recommend use-on-cd option)
+- Use the correct node version (version defined in .nvmrc): `fnm use`
+- [Install pnpm](https://pnpm.io/installation)
+- [Install Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+## Setup
+
+Fill `KOBO_TOKEN`, `FLOOD_ASSET_ID`, `DROUGHT_ASSET_ID`, `INCIDENT_ASSET_ID` in `.env.example.rc` and `.env.test.rc`.
+
+```bash
+# Install dependencies
+pnpm install
+# Create local env file
+cp .env.example.rc .env.rc
+```
+
+## Running the app
```

Comment:
just provide the link to the backend doc for the backend

Reply:
resolved